src/test/tests/databases/blueprint.py

```diff
@@ -1,5 +1,5 @@
 # ----------------------------------------------------------------------------
-#  CLASSES: nightly
+#  CLASSES: nightly, prci
```

Comment:
Your first condition for PRCI tests isn't true for blueprint: `The test typically takes less than 20 seconds to run "normally".`

Reply:
The condition is about *adding* new tests...not about anything preexisting this work.

Reply:
I see